db/sql/05_msar.sql

```diff
@@ -5627,6 +5627,49 @@ END;
 $$ LANGUAGE plpgsql;
 
 
+CREATE OR REPLACE FUNCTION msar.list_by_record_summaries(
+  tab_id oid,
+  limit_ integer,
+  offset_ integer,
+  search_ text DEFAULT NULL,
+  table_record_summary_templates jsonb DEFAULT NULL
+) RETURNS jsonb
+LANGUAGE plpgsql STABLE
+AS $$
```

Comment:
I'd like to have a documentation comment here.

Reply:
Addressed via 70f87a2ec85b7b996fbb7013a7a8e8e73fdd3845